src/PrestaShopBundle/Resources/config/routing/admin/configure/shop_parameters/order_states.yml

```diff
@@ -5,7 +5,6 @@ admin_order_states:
     _controller: PrestaShopBundle\Controller\Admin\Configure\ShopParameters\OrderStateController::indexAction
     _legacy_controller: AdminStatuses
     _legacy_link: AdminStatuses
-    _legacy_feature_flag: order_state
```

Comment:
Too early to remove this one as well, first the new page must be validated Then in a dedicated PR we will:
- remove legacy controller
- remove legacy templates
- remove `_legacy_feature_flag` attributes
- remove the feature flag from install data (and remove it in auto upgrade scripts as well)